Documentation/concepts/networking/fragmentation.rst

```diff
@@ -21,4 +21,11 @@ options:
 - ``--bpf-fragments-map-max``: Control the maximum number of active concurrent
   connections using IP fragmentation. For the defaults, see `bpf_map_limitations`.
 
+.. note::
+
+    When running Cilium with kube-proxy, even nodePort traffic is handled by
+    kube-proxy, IPv4 fragmented packets can still break due to a kernel bug where
+    route MTU is not respected for forwarded packets. It has been (`fixed <http://patchwork.ozlabs.org/project/netdev/patch/20200923201815.388347-1-zenczykowski@gmail.com/>`_)
+    and is available now in newest patch releases of all maintained kernel versions.
```

Comment:
```suggestion
    and has been backported to all maintained kernel versions.
```
Not sure if this is future-proof, maybe “to Linux 4.9 and later maintained versions” (or even 4.14)? (I think the oldest maintained is 4.4? But Cilium requires 4.9, and fragment tracking support requires at least 4.10 anyway)

Reply:
I think we should link to [the upstream commit](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=02a1b175b0e92d9e0fa5df3957ade8d733ceb6a0) (see above comment) and then folks can figure out if their kernel has it. Trying to list which kernels have it is going to be fragile.

Reply:
done